test/bootstrap_form_group_test.rb

```diff
@@ -316,7 +318,7 @@ class BootstrapFormGroupTest < ActionView::TestCase
 
     expected = <<~HTML
       <div class="mb-3 row">
-        <label class="form-label col-form-label col-sm-2 required" for="user_email">Custom Control</label>
+        <label class="form-label col-form-label col-sm-2" for="user_email">Custom Control</label>
```

Comment:
I believe this field is required. E-mail is required by the underlying model, so we want to make it a required field on the form, so this class would remain here, and you'd add the `aria-required` and `required` attributes to the `input`.

Reply:
You are right this was a bug in my code  I think I have fixed it